test/models/group_offer_scopes_test.rb

```diff
@@ -0,0 +1,101 @@
+require 'test_helper'
+
+class GroupOfferScopesTest < ActiveSupport::TestCase
+  test 'active' do
+    group_offer_active = create :group_offer, active: true
```

Comment:
It is true by default

Reply:
who cares about defaults. Where testing here, so what if the default is ever changed?